src/Resources/config/fastly_proxy.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="fos_http_cache.test.proxy_server.fastly"
+                 class="FOS\HttpCache\Test\Proxy\FastlyProxy">
```

Comment:
i don't think a FastlyProxy class exists.

we use this pattern to start a varnish / nginx instance during testing. i don't think we can do that for fastly. 

can we remove this config file and the DI method that loads it?